source/deprecations/v2.x.html.md

```diff
@@ -727,3 +676,54 @@ arr.without(NaN);         // ['a', 'b']
 
 
 Added in [PR #13553](https://github.com/emberjs/ember.js/pull/13553).
+
+### Deprecations Added in Pending Features
```

Comment:
Hehe, I did the same thing when I reviewed this commit before submitting the PR.